### PR TITLE
Add additional clarifications.

### PR DESCRIPTION
**Pull request by orthener**
## Purpose of this pull request

This pull request (PR) for updating "Marketing review documentation".
I could not get through the review, because, as it turned out, there are some rules that not described in this documentation. So decided add this points.

## Affected DevDocs pages

-  https://devdocs.magento.com/marketplace/sellers/product-descriptions.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Review by BarnyShergold — APPROVED** [review]

**Comment by keharper** [comment]
Looking for an internal reviewer to confirm.

**Comment by meker12** [comment]
Contacted Marketplace team to verify update.

File: src/marketplace/sellers/product-descriptions.md [hunk 1]
```diff
@@ -96,4 +96,4 @@ _Example 3:_
 
 ATTENTION: Use this extension at your own risk. Sensitive customer payment data is stored on your server. Merchants are expected to take every precaution that they are fully PCI certified. Please only purchase this extension if you have an environment where storing credit card information is permitted by law.
 
-[1]: https://marketplacesupport.magento.com/hc/en-us
\ No newline at end of file
+[1]: https://marketplacesupport.magento.com/hc/en-us
```
Comment (mikeweis) [inline src/marketplace/sellers/product-descriptions.md:99]: Not sure why this line (`#99`) shows up as a change; it looks identical to me! 😸

**Comment by meker12** [comment]
image

**Review by meker12 — APPROVED** [review]

File: src/marketplace/sellers/product-descriptions.md (hunk 1 above)
Comment (orthener) [reply src/marketplace/sellers/product-descriptions.md:99]: I don't know why this line highlighted ))

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Pull request merged**